### PR TITLE
fix: apply background in popover

### PR DESCRIPTION
**Pull request by leonid**
### Type of change

- [ ] Documentation (updates to the documentation, readme or JSDoc annotations)
- [ ] Bug fix (a non-breaking change that fixes an issue)
- [x] Enhancement (improving an existing functionality like performance)
- [ ] New feature (a non-breaking change that adds functionality)
- [x] Refactoring (old API is revised and supported)
- [ ] Chore (updates to the build process or auxiliary tools and libraries)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Description

apply background in popover

### CODE REVIEW

File: packages/ui/src/components/popover/popover.vue [hunk 1]
```diff
@@ -43,7 +43,9 @@
         class="acv-popover"
       >
         <AcvCard
-          class="acv-popover__content"
+          class="content"
+          background-color="white"
```
Comment (necobaka) [inline packages/ui/src/components/popover/popover.vue:47]: Why strictly white? shouldn't there be a dependency on brand or light/dark mode?

Comment (leonid) [reply packages/ui/src/components/popover/popover.vue:47]: Now we don't have dark variation. 
Its just white in Acronis ui-kit.
New mockups also don't have dark scheme.

Comment (Jerd0) [inline packages/ui/src/components/popover/popover.vue:47]: mb, is it better to use css variables instead of props?

Comment (leonid) [reply packages/ui/src/components/popover/popover.vue:47]: Yes, but Card component is not fully implemented yet

**Review by Jerd0 — APPROVED** [review]
i hope we will refactor card component

**Pull request merged**